sources/main.cpp

```diff
@@ -543,10 +597,16 @@ int main(int argc, const char** argv) try {
         }
     }
 } catch (const std::exception& error) {
-    std::cerr << "Error: " << error.what() << '\n';
+    std::cerr << "Fatal error: " << error.what() << '\n';
+    return EXIT_FAILURE;
+} catch (const Error& error) {
+    std::string description;
```

Comment:
Added an exception output for cases when `llvm::Error` is thrown by the engine (e.g., when failing to construct the command options parser).